### ISSUE TITLE
MVM: Revise name of trailer files

### ISSUE THREAD

**Issue by stscijgbot-hstdp** [spacetelescope/drizzlepac#821]
_Issue [HLA-435](https://jira.stsci.edu/browse/HLA-435) was created on JIRA by [Warren Hack](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hack):_

Trailer files generated during MVM processing need to have filenames which are consistent with the products and inputs.

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Warren Hack](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hack) on [JIRA](https://jira.stsci.edu/browse/HLA-435?focusedCommentId=494034#comment-494034):_

Drizzlepac #&#x2060;841 will also address this ticket as the changes required fixes from #&#x2060;841 as well in order to address the trailer file naming bug.

**Comment by stsci-hack** [comment]
Will be closed by #841 .

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Lisa Sherbert](https://jira.stsci.edu/secure/ViewProfile.jspa?name=sherbert) on [JIRA](https://jira.stsci.edu/browse/HLA-435?focusedCommentId=494301#comment-494301):_

I compared the trailer names above to the regex from HSTSDP-1188 and they match this regex:
```java
hst_(skycell-p\d{4}x\d{2}y\d{2})_([a-z]{3}[0-9]{0,1})_([a-z]{3,4})_([a-z]\d{3}[a-z])_([^\s]+)_([a-z]{3})
```
as well as the fits files in /ifs/archive/dev/hst/autoDataStore/input/regr_08130/data-fm-scsb/hst_skycell-p2381x05y09 do
